doc/man3/BIO_ctrl.pod

```diff
@@ -12,8 +12,7 @@ BIO_get_info_callback, BIO_set_info_callback, bio_info_cb
 
  #include <openssl/bio.h>
 
- typedef void (*bio_info_cb)(BIO *b, int oper, const char *ptr,
-                             int arg1, long arg2, long arg3);
+ typedef int (*bio_info_cb)(BIO *b, int state, int res);
```

Comment:
So I would change this to
`typedef int BIO_info_cb(BIO *b, int state, int res);`

and fix all `bio_info_cb` to `BIO_info_cb *`